excel/beginner/README.md

```diff
@@ -0,0 +1,12 @@
+name: Beginner
+
+description: Learn how to create workbooks and worksheets as well as simple and complex formulas to better analyze data.

```

Comment:
Maybe instead of "simple and complex" we can say "various" or "different"? Or maybe just omit that term completely?

```suggestion
description: Learn how to create workbooks and worksheets and use formulas to better analyze data.
```

Reply:
A simple formula is A1 + A2 or 1+ A2, etc.. Any formula which has only 1 operation is considered a simple formula, whereas any formula with 2 or more is considered complex. But sure, we can omit it